src/Application.js

```diff
@@ -94,6 +94,21 @@ import ConfigurationManager from './ConfigurationManager'
 import Watcher from './Watcher'
 import Logger from './Logger'
 
+var _defaultConfig = {
+  siteName: 'Nxus App',
+  host: 'localhost',
+  baseUrl: 'localhost:3000'
+}
+
+_.deepClone = function(obj) {
```

Comment:
Good catch - I tried underscore.deepClone but couldn't get it to work.  Underscore.deep works nicely.